Implementation read/write of VOTable with Parquet serialization (#15281)
astropy/astropy: docs/io/votable/index.rst

  supported by the `astropy.io.votable` writer, since it requires
  writing multiple files.

> - `PARQUET`
>   stores the data in an external PARQUET file, similar to FITS serialization.
>   Reading and writing is fully supported by the `astropy.io.votable` writer and
>   the `parse()` reader. Works for absolute and relative paths. This serialization
>   can be used by setting the table ``format`` attribute to `votable.parquet`.
> 

Reviewer: Single backticks mean sphinx will try to linkify the words, and will fail as no reference will be found for most of these:
```suggestion
- ``PARQUET``
  stores the data in an external PARQUET file, similar to FITS serialization.
  Reading and writing are fully supported by the `astropy.io.votable` writer and
  the `parse()` reader. Works for absolute and relative paths. This serialization
  can be used by setting the ``'format'`` argument to ``'votable.parquet'``.

```

  supported by the `astropy.io.votable` writer, since it requires
  writing multiple files.

- ``PARQUET``
  stores the data in an external PARQUET file, similar to FITS serialization.
  Reading and writing is fully supported by the `astropy.io.votable` writer and
  the `astropy.io.votable.parse` reader. The parquet file can be
  referenced with either absolute and relative paths. The parquet
  serialization can be used as part of the unified Table I/O (see next

>   section), by setting the ``'format'`` argument to ``'votable.parquet'``.

Reviewer 2: ```suggestion
  section), by setting the ``format`` argument to ``'votable.parquet'``.
```
